ng-dev/release/publish/external-commands.ts

```diff
@@ -82,6 +82,40 @@ export abstract class ExternalCommands {
     }
   }
 
+  /**
+   * Invokes the `ng-dev release npm-dist-tag delete` command in order to delete the
+   * NPM dist tag for all packages in the checked-out version branch.
+   */
+  static async invokeDeleteNpmDistTag(projectDir: string, npmDistTag: NpmDistTag) {
+    // Note: We cannot use `yarn` directly as command because we might operate in
+    // a different publish branch and the current `PATH` will point to the Yarn version
+    // that invoked the release tool. More details in the function description.
+    const yarnCommand = await resolveYarnScriptForProject(projectDir);
```

Comment:
Since we always do this in these commands, should we just do it as a file level constant?

Reply:
I think no. The risk is high that the Yarn path would change if we operate in a different branch at that time.